packages/api/src/lib/prepareParams.js

```diff
@@ -27,6 +24,61 @@ function isEmpty(obj) {
   return [Object, Array].includes((obj || {}).constructor) && !Object.entries(obj || {}).length;
 }
 
+function processFile(paramName, file) {
+  if (typeof file === 'string') {
+    // In order to support relative pathed files, we need to attempt to resolve them.
+    const resolvedFile = path.resolve(file);
+    if (!fs.existsSync(resolvedFile)) {
```

Comment:
Maybe this should be awaited instead of blocking using existsSync? Though I dont actually know what the difference is between `existsSync()` and `await exists()`

Reply:
Yeah good catch, I also don't need to wrap `getStream` in a promise as well because it's already one.